Formula/zrok.rb

```diff
@@ -0,0 +1,41 @@
+require "language/node"
+
+class Zrok < Formula
+  desc "Geo-scale, next-generation sharing platform built on top of OpenZiti"
+  homepage "https://zrok.io"
+  url "https://github.com/openziti/zrok/archive/refs/tags/v0.3.6.tar.gz"
+  sha256 "2543bffb999428abf20ce0f2dec3d63e32ffb624a51255bcc3e76356905b4135"
+  license "Apache-2.0"
+  head "https://github.com/openziti/zrok.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "node" => :build
+
+  FULLHASH = "50c5a48891b3f53825ae0714e5ab58345389d0fd".freeze
```

Comment:
what is this?

Reply:
We were using the full commit hash in our build to explicitly version it. If it's preferable to use brew as below then this will get removed.